# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

GraspHandleActionGoal action_goal
GraspHandleActionResult action_result
GraspHandleActionFeedback action_feedback
